cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -2122,6 +2122,13 @@
         (ones Reg (load_imm12 -1)))
   (value_reg (gen_select_reg (IntCC.Equal) zero input zero ones))))
 
+(decl normalize_fcvt_from_int (ValueRegs Type ExtendOp) ValueRegs)
```

Comment:
Can we move this next to the other `normalize` functions? Having it in the middle of the `lower_bmask` rules makes them somewhat difficult to read.

Reply:
sure.